providers/edge3/src/airflow/providers/edge3/plugins/edge_executor_plugin.py

```diff
@@ -227,6 +227,26 @@ def change_maintenance_comment(self, worker_name: str):
     RUNNING_ON_APISERVER = "gunicorn" in sys.argv[0] and "airflow-webserver" in sys.argv
 
 
+def _get_base_url_path(path: str) -> str:
+    """Construct URL path with webserver base_url prefix."""
+    base_url = conf.get("api", "base_url", fallback="/")
+    # Extract pathname from base_url (handles both full URLs and path-only)
+    if base_url.startswith(("http://", "https://")):
+        from urllib.parse import urlparse
+
+        base_path = urlparse(base_url).path
+    else:
+        base_path = base_url
+
+    # Normalize paths: remove trailing slash from base, ensure leading slash on path
+    base_path = base_path.rstrip("/")
+    if not path.startswith("/"):
+        path = "/" + path
+
+    # Combine paths
```

Comment:
I assume this is not needed, we have a private method and all references are consistent below.
```suggestion
```

Reply:
I seem to need this, without it im hitting issues

Reply:
<img width="1272" height="692" alt="Image" src="https://github.com/user-attachments/assets/915a1432-4f4d-47ea-b7c1-58851e32a1d6" />

Reply:
Okay, you are right. But I was mainly referring to the `path` as provided via parameter which is consistent:
```suggestion
    # Normalize paths: remove trailing slash from base, ensure leading slash on path
    base_path = base_path.rstrip("/")
```